src/main/g8/src/main/scala/$package$/Main.scala

```diff
@@ -0,0 +1,19 @@
+package $package$
+
+import java.io.IOException
+
+import zio.{ App, ExitCode, URIO, ZEnv, ZIO }
+import zio.console.{ getStrLn, putStrLn, Console }
+
+object Main extends App {
+
+  val app: ZIO[Console, IOException, Unit] =
+    for {
+      _    <- putStrLn("What is your name?")
+      name <- getStrLn
+      out  <- putStrLn(s"Hello $name!")
```

Comment:
`$` needs to be escaped in templates

Reply:
`sbt g8Test` generate properly template

Reply:
Done: https://github.com/ScalaConsultants/zio-dotty-quickstart.g8/pull/1/commits/0c32a87b4ae6de3d55344d61e05abeac8fc1887b

Looks like `sbt g8Test` do not complain about that :) It is better to test using `g8`